oas-generator-spi/oas-generator-asciidoctor/src/main/resources/freemarker/openapi.ftlh

```diff
@@ -10,29 +10,18 @@
 <#--
 
 -->
-<#if standalone>
 <@section title=openapi.info.title level=1/>
 <#if openapi.info.contact??>
 ${openapi.info.contact.name} <${openapi.info.contact.email}>
 </#if>
+<#if standalone>
 :last-update-label: Generated by `OAS Generator` based on OpenAPI Specification (v${openapi.openapi}) on
 :revnumber: ${openapi.info.version}
 :icons: ${attributeIcons}
 :toc: left
 :toclevels: 3
```

Comment:
The closing if-block of the standalone part should be moved below the asciidoc attributes. Otherwise the description will only be added for standalone files.

```suggestion
:toclevels: 3
</#if>

<#if openapi.info.description??>
${openapi.info.description}
</#if>
```

You could verify this behaviour in the unit test you created.

Reply:
Added more tests, moved closing if around.